product-scenarios/2-Bridging-systems-that-communicate-in-different-protocols/pom.xml

```diff
@@ -32,9 +32,126 @@
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <argLine>-Xms512m -Xmx1024m -XX:MaxPermSize=128m</argLine>
+                            <disableXmlReport>false</disableXmlReport>
+                            <parallel>false</parallel>
+                            <suiteXmlFiles>
+                                <suiteXmlFile>${basedir}/src/test/resources/testng.xml</suiteXmlFile>
+                            </suiteXmlFiles>
+                            <systemProperties>
+                                <property>
+                                    <!-- Common resource directory (eg: Key stores, certificates, etc.)-->
+                                    <name>common.resources.dir</name>
+                                    <value>
+                                        ${basedir}/../../resources/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- testcase specific  resources directory-->
+                                    <name>test.resources.dir</name>
+                                    <value>
+                                        ${basedir}/src/test/resources/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- Directory containing carbon applications -->
+                                    <name>test.resources.carbonApplications.dir</name>
+                                    <value>
+                                        ${basedir}/../../02-SynapseConfigProject/02-synapseConfigCompositeApplication/target/
+                                    </value>
+                                </property>
+                                <property>
+                                    <!-- Related carbon applications (comma separated) to deploy -->
+                                    <name>test.resources.carbonApplications.list</name>
+                                    <value>
+                                        02-synapseConfigCompositeApplication_1.0.0
+                                    </value>
+                                </property>
+                                <property>
+                                    <name>usedefaultlisteners</name>
+                                    <value>false</value>
+                                </property>
+                            </systemProperties>
+                            <workingDirectory>${basedir}/target</workingDirectory>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>profile_490</id>
```

Comment:
Do we need to maintain a separate profile for EI 4.9.0 in scenario 2 as well?

Reply:
no need, let's remove

Reply:
done